Kernel/Bus/PCI/API.cpp

```diff
@@ -29,69 +29,69 @@ DeviceIdentifier get_device_identifier(Address address)
 
 HardwareID get_hardware_id(Address address)
 {
-    return { read16(address, PCI_VENDOR_ID), read16(address, PCI_DEVICE_ID) };
+    return { read16(address, to_underlying(PCI::RegisterOffset::VENDOR_ID)), read16(address, to_underlying(PCI::RegisterOffset::DEVICE_ID)) };
```

Comment:
Hmm, I'm not sure I like where this is going.
This `to_underlying()` business is extremely verbose and makes the code much less readable.

Reply:
I see. I removed almost all `to_underlying` calls right now (without sacrificing quality really), should be good now I think :)